ui/apps/platform/src/Containers/VulnMgmt/List/Cves/VulnMgmtListCves.js

```diff
@@ -391,11 +391,9 @@ const VulnMgmtCves = ({
             return;
         }
 
-        cveNames.forEach((cve) => {
-            analyticsTrack({
-                event: GLOBAL_SNOOZE_CVE,
-                properties: { type, cve, duration },
-            });
+        analyticsTrack({
+            event: GLOBAL_SNOOZE_CVE,
+            properties: { type, duration },
         });
```

Comment:
Since we do not want to track individual CVEs, there is no need to send a separate event for each CVE in the snooze action.

Reply:
I don't think it is terribly interesting, the main use case for this event was so that we could get insight into whether or not users are using the global snooze for Node and Platform CVEs at all. The analytics here should disappear once VM 1.0 is fully deprecated.